lib/XML/LibXML/Document.pm

```diff
@@ -45,7 +46,10 @@ method doctype is aka<type> {
     #~ This read-only property is an object that implements the DOMImplementation interface.
 
 #| This read-only property is an object that implements the Element interface.
-method documentElement is aka<root> {
+method documentElement 
```

Comment:
I think we can use them since aliases are also done in p5 XML::LibXML. I think that we should try to go for as much of a drop in replacement where it makes sense. And document well where things deviate. I've been using the XS code from p5's XML::LibXML as a guide and aliasing is heavily used, there.